Header header

geometry_msgs/Pose  pose
geometry_msgs/Twist   velocity
quadrotor_msgs/ControlCommand   control_command_mpc
quadrotor_msgs/Trajectory  horizon_trajectory
float64   execution_time
float64   preparation_time
float64   objective_function_value
float64   kkt_value
int8      solver_status
geometry_msgs/Wrench    external_wrench
bool      interaction
bool      sliding_x
bool      sliding_y
bool      sliding_yaw
geometry_msgs/Twist   interaction_reference_velocity
